src/index.html

```diff
@@ -20,8 +20,347 @@
       href="styles/index.scss"
     />
   </head>
-
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a
+        href="index.html"
+        class="logo"
+      >
+        <img
+          src="images/logo.png"
+          alt="Moyo Logo"
+        />
+      </a>
+      <nav class="container">
+        <ul class="navigation">
+          <li>
+            <a
+              href="#appl"
+              class="navigation__item navigation__item--apple navigation__item--is-active"
+            >
+              APPLE
+            </a>
+          </li>
+          <li>
+            <a
+              href="#sams"
+              class="navigation__item navigation__item--samsung"
+            >
+              SAMSUNG
+            </a>
+          </li>
+          <li>
+            <a
+              href="#boxsmart"
+              class="navigation__item navigation__item--smartphones"
+            >
+              SMARTPHONES
+            </a>
+          </li>
+          <li>
+            <a
+              href="#laptops"
+              class="navigation__item navigation__item--laptops"
+              data-qa="nav-hover"
+            >
+              LAPTOPS & COMPUTERS
+            </a>
+          </li>
+          <li>
+            <a
+              href="#gadgets"
+              class="navigation__item navigation__item--gadgets"
+            >
+              GADGETS
+            </a>
+          </li>
+          <li>
+            <a
+              href="#tablet"
+              class="navigation__item navigation__item--tablets"
+            >
+              TABLETS
+            </a>
+          </li>
+          <li>
+            <a
+              href="#photo"
+              class="navigation__item navigation__item--photo"
+            >
+              PHOTO
+            </a>
+          </li>
+          <li>
+            <a
+              href="#videonp"
+              class="navigation__item navigation__item--video"
+            >
+              VIDEO
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="card-menu">
+      <div
+        class="card"
+        data-qa="card"
+      >
+        <div>
+          <img
+            class="card__imagep"
```

Comment:
The class name `card__imagep` seems to be a typo. It should be `card__image` to maintain consistency with the BEM naming convention.